articles/dev-spaces/quickstart-java.md

```diff
@@ -31,8 +31,8 @@ In this guide, you will learn how to:
 - A Kubernetes cluster running Kubernetes 1.10.3 or later, in the EastUS, CentralUS, WestUS2, WestEurope, CanadaCentral, or CanadaEast region, with **Http Application Routing** enabled.
 
     ```cmd
-    az group create --name MyResourceGroup --region <region>
-    az aks create -g MyResourceGroup -n myAKS --location <region> --kubernetes-version 1.11.2 --enable-addons http_application_routing
+    az group create --name MyResourceGroup --location <region>
```

Comment:
This is a bug in the .NET Core with VS Code and Node.js quickstarts as well - can you fix there too? Thanks!

Reply:
Already done :)
https://github.com/MicrosoftDocs/azure-docs/pull/16713
https://github.com/MicrosoftDocs/azure-docs/pull/16714